StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet+API.swift

```diff
@@ -73,6 +62,13 @@ extension PaymentSheet {
         switch paymentOption {
         // MARK: - Apple Pay
         case .applePay:
+            guard let intent = confirmableIntent.asIntent else {
+                let message =
+                    "Cannot present Apple Pay without an Intent"
+                assertionFailure(message)
+                completion(.failed(error: PaymentSheetError.unknown(debugDescription: message)))
+                return
+            }
```

Comment:
Does this mean Apple Pay isn't supported?

Reply:
See Link comment.